keras_cv/visualization/plot_image_gallery.py

```diff
@@ -178,8 +176,14 @@ def plot_image_gallery(
             current_axis.margins(x=0, y=0)
             current_axis.axis("off")
 
+    if path is None and show is None:
+        #If path is None and show is None
+        # Default to showing the image
+        return fig
+    
     if path is None and not show:
```

Comment:
We can probably drop this conditional now since the one above will cover the same case

Reply:
@Zekrom-7780 it looks like this never got removed. Other than that this LGTM